packages/bpk-mixins/src/mixins/_badges.scss

```diff
@@ -32,14 +32,14 @@
 
 @mixin bpk-badge {
   display: inline-flex;
-  padding: $bpk-badge-padding-y $bpk-badge-padding-x;
+  padding: .175rem $bpk-badge-padding-x;
```

Comment:
This isn't a token value now, why?

Reply:
It is for the correct spacing that matches the Sketch libraries and we don't have a token value here for this one

Reply:
Hmmmmmmmmmmmmmmmmmmmmmmmmmmmmm.